resource/traversers/dfu_impl.cpp

```diff
@@ -558,6 +558,16 @@ int dfu_impl_t::dom_dfv (const jobmeta_t &meta, vtx_t u,
         m_err_msg += strerror (errno);
         m_err_msg += ".\n";
         goto done;
+    } else if ((meta.job_type == "elastic") && (*m_graph)[u].schedule.elastic_job) {
```

Comment:
Super minor nit: not sure about the proper style here but elsewhere it seems like there is a space between the paren.s

```suggestion
    } else if ( (meta.job_type == "elastic") && (*m_graph)[u].schedule.elastic_job) {
```

Reply:
Good catch, I'll make that change.